src/main/resources/com/rultor/agents/req/merge.sh

```diff
@@ -20,6 +20,12 @@ while [ $(git show-branch "${BRANCH}" 2>/dev/null | wc -l) -gt 0 ]; do
 done
 
 git checkout -B "${BRANCH}" "fork/${fork_branch}"
+git fetch origin
+if git log --name-status --oneline HEAD..origin/${head_branch} | grep ".rultor.yml"; then
+  echo "Since original repository has different version of .rultor.yml file you have to sync your fork with the upstream repository."
+  exit 1
+fi
```

Comment:
@original-brownbear @mkordas Yes, the best way will be use merged version of `.rultor.yml`. But there is a deadlock: to merge `.rultor.yml` we need to have merged `.rultor.yml` file. This task is not an easy one to solve it inside rultor (in case, if we are not constructing AI here). I think it is better to move merging (and solving possible conflicts) to PR author side. Yes it can be noisy enough (if `.rultor.yml` is changed very frequently) but I don't see any better solution here.